src/_base/helm/qa/values.yaml.twig

```diff
@@ -1,7 +1,8 @@
 {{ @('workspace.name') }}:
-  environment:
-    APP_HOST: {{ @('pipeline.qa.hostname') }}
   resourcePrefix: {{ @('pipeline.qa.resourcePrefix') | json_encode | raw }}
+  docker:
+    services:
+{{ deep_merge_to_yaml([@('services'), @('pipeline.qa.services')], 2, 6) | raw }}       
```

Comment:
@('services') is already available in the app chart values, so doesn't need to be specified here. I'm a bit confused how this original isn't what I had intended, which I wrote as:
```
{{% if @('pipeline.qa.services') %}
  docker:
    services:
{{ to_nice_yaml(@('pipeline.qa.services'), 2, 6) | raw }}  
{{% endif %}
```